Formula/fabric.rb

```diff
@@ -12,8 +12,6 @@ class Fabric < Formula
     sha256 "bb39fa519461b1eb648b071a8cd06e13f4c10cda1fcd01b02a44be422019c37a" => :yosemite
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
I see in https://github.com/Homebrew/brew/blob/master/docs/Installation.md it says
> 10.10 or higher is recommended. 10.5 - 10.9 are supported on a best-effort basis.

So I'll get rid of this commit.